libyul/optimiser/StructuralSimplifier.h

```diff
@@ -46,6 +48,7 @@ class StructuralSimplifier: public DataFlowAnalyzer
 	void simplify(std::vector<Statement>& _statements);
 	bool expressionAlwaysTrue(Expression const& _expression);
 	bool expressionAlwaysFalse(Expression const& _expression);
+	boost::optional<dev::u256> hasLiteralValue(Expression const& _expression) const;
```

Comment:
Shouldn't this go into the utils?

Reply:
It does use a member variable m_values though

Reply:
Ok, didn't see that.